bin/pyenv-pip-update

```diff
@@ -27,7 +27,11 @@ then # If no argument is given, update libraries using pip
             conda update --all
         elif pip >/dev/null 2>&1
         then
-            pip list --outdated | awk '{if ($1 != "Package" && $1 !~ /^-+$/) print $1}' | xargs pip install -U
+            stale_packages="$(pip --no-python-version-warning list --format=freeze --outdated | awk -F '==' '{printf "%s ", $1}')"
```

Comment:
You need to decide whether to use `--no-python-version-warning` depending on the version of pip becouse this option isn't available before pip 20.0.

Reply:
Fixed